### DISCUSSION TITLE
Ungroup settings

### DISCUSSION THREAD

**Discussion by batal131** [jilleb/MQB-FPA#43] [Exploration and research]
In most datasets Engine and Gearbox are grouped as "Drive". Did anyone manage to ungroup them? I want to have them as separate options.

Or maybe someone have working example ("before" and "after") with ungrouping any other settings?

**Comment by jilleb** [comment]
Yes, I was able to do it, it's quite easy. I'll show an example of a custom dataset I'm working on:
[a lot of profiles.zip](https://github.com/jilleb/MQB-FPA/files/7090950/a.lot.of.profiles.zip)

**Reply by batal131** [reply]
Thanks. Almost the same as I did with mine. But it does not work. I'll try again today.

I have small question about controls after ungrouping.
At 4CDh there is list of controls with groups. Initially I have 14 (0-13) controls here. After ungrouping I have 19 controls (0-18)
At 9EBh I have 19 controls from factory.

Question - should 4CDh and 9EBh be equal? or it's ok that they have not the same order?

That's because after ungrouping 4CDh I've added "new" controls at the end.

(the only thing I'm trying to do is remove groups. initially I have 2 groups: 5 + 2)

**Reply by batal131** [reply]
Actually I'm talking about this:
Controls are the same, but order is different. 
![12345](https://user-images.githubusercontent.com/964702/131694750-64020cff-b1c3-4801-a5ad-5d72c1646d84.png)

**Reply by tru1** [reply]
> Yes, I was able to do it, it's quite easy. I'll show an example of a custom dataset I'm working on: [a lot of profiles.zip](https://github.com/jilleb/MQB-FPA/files/7090950/a.lot.of.profiles.zip)

Hi,

Do you know if its possible to split Group 1? I want to take away Engine from Group 1,but leave other setting in it.

Group in standard config:
Group 1 (Drive): Gearbox
Group 1 (Drive): Engine
Group 1 (Drive): Start/Stop System
Group 1 (Drive): DSG Coasting
Group 1 (Drive): Freilauf_DefaultON

I want to have:
Group 1 (Drive): Gearbox
Group 1 (Drive): Start/Stop System
Group 1 (Drive): DSG Coasting
Group 1 (Drive): Freilauf_DefaultON

And Engine not in the group1

**Comment by batal131** [comment]
After changing order ( 4CDh and 9EBh are equal) - everything works. Thanks!
